fakebook/src/app/services/post.service.ts

```diff
@@ -32,10 +32,11 @@ export class PostService {
     return this.http.get<Post[]>(`${this.url}/user/${email}`);
   }
 
-  delete(postId: number): Observable<void> {
-    return this.http.delete<void>(`${this.url}/${postId}`);
-  }
   update(post: Post): Observable<Post> {
     return this.http.put<Post>(`${this.url}/${post.id}`, post);
   }
+  delete(postId: number): null {
+    this.http.delete<void>(`${this.url}/${postId}`).subscribe();
+    return null;
+  }
```

Comment:
why behave unlike all the other methods here?

Reply:
because I was having trouble getting the test to work returning an observable and then subscribing to it.  karma was being a pain, and we didn't do anything with the return anyways.  I agree it doesn't look as clean though, I can switch it back to returning an observable and figure out how to mock return one in the tests the rest of today/this weekend

Reply:
you can have the test observable just be an observable that sends `undefined`.